src/GraphQL/Validation/DocumentValidator.cs

```diff
@@ -49,7 +49,7 @@ public class DocumentValidator : IDocumentValidator
             NoUndefinedVariables.Instance,
             NoUnusedVariables.Instance,
             UniqueVariableNames.Instance,
-            KnownDirectives.Instance,
+            KnownDirectivesInAllowedLocations.Instance,
```

Comment:
I renamed `KnownDirectives` to `KnownDirectivesInAllowedLocations` to more accurately reflect the meaning of the validation rule. In GraphQL.NET this validation rule checks two requirements from the specification and reports two errors with different numbers - `5.7.1` and `5.7.2`.

Reply:
Answers my question from above.  👍🏼